examples/pom.xml

```diff
@@ -221,14 +221,21 @@
       <groupId>com.github.cukespace</groupId>
       <artifactId>cukespace-core</artifactId>
       <version>${project.version}</version>
-      <scope>test</scope>
       <exclusions>
         <exclusion>
           <groupId>com.google.guava</groupId>
           <artifactId>guava</artifactId>
         </exclusion>
       </exclusions>
+      <scope>test</scope>
     </dependency>
+    
+    <dependency>
```

Comment:
The error is Archive for required library: 'C:/Users/CFO/.m2/repository/net/sourceforge/cssparser/cssparser/0.9.9/cssparser-0.9.9.jar' in project 'cukespace-examples' cannot be read or is not a valid ZIP file, which is weird because maven should download it for me, maybe it has to do with some bad download, let me recheck